test/export/InstanceExporter.test.ts

```diff
@@ -10311,7 +10342,10 @@ describe('InstanceExporter', () => {
       afterEach(() => {
         // None of the test expect warnings or errors. All should be clean.
         expect(loggerSpy.getAllLogs('warn')).toHaveLength(0);
-        expect(loggerSpy.getAllLogs('error')).toHaveLength(0);
+        expect(loggerSpy.getAllLogs('error')).toHaveLength(1);
+        expect(loggerSpy.getMessageAtIndex(0, 'error')).toMatch(
+          /Multiple FSH entities created with name/
+        );
```

Comment:
I'm a little confused by this. Why is this error appearing for these tests? There don't appear to be any entities with the same names defined.

Reply:
This error is occurring for a different reason than below. 

Stepping through this test, I see that when export is called in this group of tests, it adds the profile (CarProfile) and the logical (Car) at the same time, so when calling export for the Logical, its already in the this.pkg.FshMap when it checks for it, and throws the error.